src/interfaces.jl

```diff
@@ -591,27 +591,27 @@ function plot!(scene::Combined, ::Type{PlotType}, attributes::Attributes, args..
     push!(scene.plots, plot_object)
     scene
 end
-
-
-
-
-
+function apply_camera!(scene::Scene, cam_func)
+    if cam_func in (cam2d!, cam3d!, campixel!, cam3d_cad!)
```

Comment:
Just putting a reminder here that we should probably mention this line in devdocs somewhere...

Reply:
Mentioned in devdocs.